ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -113,53 +82,58 @@ message CreatedEvent {
   ContractMetadata metadata = 10;
 }
 
-// View of a create event matched by an interface filter.
-message InterfaceView {
+// Records that a contract has been transferred-out, and it becomes unusable on the source domain
+message TransferredOutEvent {
 
-  // The interface implemented by the matched event.
+  // The ID of the transfer out event. This needs to be used as an input for a transfer-in TransferCommand.
+  // Must be a valid LedgerString (as described in ``value.proto``).
   // Required
-  Identifier interface_id = 1;
+  string transfer_out_id = 1;
 
-  // Whether the view was successfully computed, and if not,
-  // the reason for the error. The error is reported using the same rules
-  // for error codes and messages as the errors returned for API requests.
+  // The ID of the created contract.
+  // Must be a valid LedgerString (as described in ``value.proto``).
   // Required
-  google.rpc.Status view_status = 2;
+  string contract_id = 2;
 
-  // The value of the interface's view method on this event.
-  // Set if it was requested in the ``InterfaceFilter`` and it could be
-  // sucessfully computed.
-  // Optional
-  Record view_value = 3;
+  // The ID of the source domain
+  // Must be a valid domain ID
+  // Required
+  string source = 3;
+
+  // The ID of the target domain
+  // Must be a valid domain ID
+  // Required
+  string target = 4;
 
+  // Transfer-in exclusivity
+  // Before this time, only the initiator of the transfer-out can initiate the transfer-in
```

Comment:
Do you want to include the initiator of the transfer-out in the message so that an application can meaningfully interpret this field? (I.e., know when the application can submit a transfer-in)

Reply:
It would also be good to have the initiator of the transfer-out to debug contention issues due to transfers.